lib/key-generators/index.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
-  "file-hash": require('./file-hash')
+  "file-hash": require('./file-hash'),
+  "version-commit": require('./version-commit')
```

Comment:
README update reports `git-commit` is an option, but this only adds `version-commit`